onnxscript/tests/function_libs/torch_lib/ops_test_common.py

```diff
@@ -473,6 +471,8 @@ def _capture_graph_and_evaluate_torch_script_evaluator(function: Callable, args,
                         input.value = subarg
                         sequence_input.append(input)
                         ort_inputs[input_name] = subarg
+                    else:
+                        sequence_input.append(subarg)
```

Comment:
Let's put comment here explaining why this is needed in case we forget?

Reply:
Done

Reply:
Good point. I wonder if there are things we don't expect that can sneak in? Nested lists?

Reply:
No idea. Just in case.

Reply:
SG. Created https://github.com/microsoft/onnxscript/issues/892 for now.